pkg/oauth/apis/oauth/validation/validation.go

```diff
@@ -139,6 +165,15 @@ func ValidateClient(client *oauthapi.OAuthClient) field.ErrorList {
 		allErrs = append(allErrs, ValidateScopeRestriction(restriction, field.NewPath("scopeRestrictions").Index(i))...)
 	}
 
+	if client.AccessTokenTimeoutSeconds != nil {
```

Comment:
Want negative check here as well.

Reply:
And same checks on update too I believe.

Reply:
yeah about negative, update automatically checks for anything doen at create